package-lock.json

```diff
@@ -42,7 +42,7 @@
         "node": ">=14.0.0"
       },
       "peerDependencies": {
-        "ion-js": "^4.3.1",
+        "ion-js": "^4.0.0",
```

Comment:
Why are we regressing `ion-js` version?

Reply:
We didn't regress `ion-js` from the latest driver https://github.com/awslabs/amazon-qldb-driver-nodejs/blob/v2.2.0/package.json#L41. Just wanted to revert a wrong commit from the previous change. I think we need to keep the version range for minimum version requirement.